frameworks/elastic/src/main/dist/elasticsearch.yml

```diff
@@ -42,7 +42,7 @@ node.ingest: {{INGEST_ENABLED}}
 
 # https://www.elastic.co/guide/en/elasticsearch/reference/6.8/security-settings.html
 {{#SECURITY_ENABLED}}
-xpack.security.enabled: true
+xpack.security.enabled: {{XPACK_SECURITY_ENABLED}}
```

Comment:
Should `xpack.security.enabled` be `false` when Search Guard is enabled?

Reply:
yes, otherwise elastic will not start

Reply:
`xpack.security.enabled: true` and enabled searchguard leads to error

Reply:
Yep. Since we're not doing any [configuration validation](https://github.com/mesosphere/dcos-commons/blob/e9edcdb19fd2ef7058862e1fd51399666aadb49f/sdk/scheduler/src/main/java/com/mesosphere/sdk/config/validate/ConfigValidationError.java) to make sure `elasticsearch.xpack_security_enabled` and `elasticsearch.searchguard_enabled` are set to `true` at the same time, we should probably call that out in the documentation.